conf/test_full.config

```diff
@@ -23,7 +23,6 @@ params {
     variable_mods = 'Oxidation (M),Carbamidomethyl (C)'
 
     // Pipeline settings
-    filter_mzml      = true
```

Comment:
Can this parameter be removed?

Reply:
I think so.. See https://github.com/nf-core/mhcquant/pull/379#issuecomment-2991056688
@marissaDubbelaar PR is ready to review :)